tests/pages.test.js

```diff
@@ -1,43 +1,67 @@
-import { loadMovie } from "../src/movies.js";
+import { describe, expect, test } from "@jest/globals";
+import { marked } from "marked";
+import { loadMovie, loadMovies } from "../src/movies.js";
 import request from "supertest";
 import app from "../src/app.js";
 
-test("hompage shows list of movies", async () => {
-	const response = await request(app)
+
+
+describe("Integrationtests", () => {
+
+	test("First page shows title of all movies", async () => {
+		const response = await request(app)
 		.get("/")
 		.expect(200)
 		.expect("Content-Type", "text/html; charset=utf-8");
 
-	expect(response.text.includes("Shawshank")).toBeTruthy();
-});
+		const movieList = await loadMovies();
+
+		movieList.data.forEach((movie) => {
+			expect(response.text.includes(movie.attributes.title)).toBeTruthy();
+		});
+	});
+
+	test("Shows correct information on each individual movie page", async () => {
+		const movieList = await loadMovies();
+
+		for (let i=1; i<=movieList.data.length; i++) {
+			const response = await request(app)
+				.get("/movie/" + i)
```

Comment:
Here you are loading a video based on it's _index_, which may or may not be the same as it's ID.